app/routes.py

```diff
@@ -1 +1,317 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint
+from os import abort
+from app import db
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from sqlalchemy import asc, desc
+import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix = "/tasks")
+goals_bp = Blueprint("goals_bp", __name__, url_prefix = "/goals")
+
+#CREATE 
+#task post route
+@tasks_bp.route("", methods=["POST"])
+def create_one_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {
+            "details": "Invalid data"
```

Comment:
Tip: It's helpful for debugging if you list why the data is invalid:

```suggestion
            "details": "Invalid data, 'title' is required."
```